### DISCUSSION TITLE
Issues Loading to Excel and Pulling 2022 League Data

### DISCUSSION THREAD

**Discussion by stormingorman5** [joeyagreco/leeger#109] [General]
Hello Joey 

First of thank you for posting this, this is great content. Im admittedly a beginner with Python but want to start learning it for work and was going to use this as my first resource. I got the entirety of the setup working but I cant load the leagues to an Excel file? In your step-by-step it says to slightly alter the code to include the call function, and then to make sure to pass into the function call to load into Excel. I dont know entirely what that means. Do I just add the "from leeger.util.excel import leagueToExcel" under the 2 functions at the top? Or do I repeat the Excel step underneath all of the first loads? I've tried several ways and cant seem to get it to work. Im also getting the same error that tprusisz is referencing here:

https://github.com/joeyagreco/leeger/issues/108

I also uploaded my script in a .txt file for reference. Would really appreciate any help! 
[my code.txt](https://github.com/joeyagreco/leeger/files/12591918/my.code.txt)

[](url)

**Comment by joeyagreco** [comment]
This should be resolved by this: https://github.com/joeyagreco/leeger/releases/tag/v2.5.4

Make sure to run `python -m pip install leeger --upgrade` and ensure you have leeger version 2.5.4 (`python -m pip show leeger`)

If you are still having issues please let me know.

**Comment by stormingorman5** [comment]
Ok I was able to update to 2.5.4 and its looking better. Now Im getting the error "Week 17 has 2 championship matches. maximum is 1 but I checked league history and its just the consolation matchup? 

Also Im still not able to get the Excel output to work? It just creates a .yahoofantasy document in the library. Is it supposed to create an Excel file as well if working correctly?

**Reply by joeyagreco** [reply]
Interesting.

The error you're getting definitely is caused by how the league loader is getting this.

There is a line of code that checks for consolation matchups [here](https://github.com/joeyagreco/leeger/blob/a1ad90e273d44a3c5c5add6fc3eb5117b2fedec1/leeger/league_loader/YahooLeagueLoader.py#L223) which is somehow not working (weird)

The .yahoofantasy document is expected; it's used for tokens during the Yahoo authentication process. You can ignore that file.

The Excel file will not be created if validation fails (which is happening in this case).

1. You can disable validation to get a slightly inaccurate Excel sheet by passing `validate=False` into the loadLeague() method.
2. In order to debug this, I would need to hop on a call with you and step through the code for your specific use case to see what's going on. I've done this before with other people who had issues with Yahoo in order to resolve issues. If this is something you're willing to do, just email me at `joeyagreco@gmail.com` and we can set up a time to do this.

**Comment by joeyagreco** [comment]
Fixed in [v2.5.5](https://github.com/joeyagreco/leeger/releases/tag/v2.5.5)
